tests/man.test

```diff
@@ -219,904 +219,6 @@ map_values(.+1)
 {"a": 1, "b": 2, "c": 3}
 {"a": 2, "b": 3, "c": 4}
 
-pick(.a, .b.c, .x)
```

Comment:
I'm surprised to see all these deletions.

Reply:
They are all duplicate lines afaict

Reply:
Why yes, they are.  LGTM then!